Makefile

```diff
@@ -121,9 +121,6 @@ test-diamond-e2e:
 test-sideinputs-e2e:
 test-%:
 	$(MAKE) cleanup-e2e
-	$(MAKE) deploy-nats
-	$(MAKE) deploy-kafka
-	$(MAKE) deploy-redis
```

Comment:
@whynowy 
While reviewing the PR I realized that the deployment of Redis wasn't controlled via the Makefile at all. Right now Sam removed the deploy-redis and it still goes forth. 
It is actually deployed for all tests in the SetupSuite() in `test/fixtures/e2e_suite.go`
```
// Create Redis resources used for sink data validation.
deleteCMD := fmt.Sprintf("kubectl delete -k ../../config/apps/redis -n %s --ignore-not-found=true", Namespace)
s.Given().When().Exec("sh", []string{"-c", deleteCMD}, OutputRegexp(""))
createCMD := fmt.Sprintf("kubectl apply -k ../../config/apps/redis -n %s", Namespace)
s.Given().When().Exec("sh", []string{"-c", createCMD}, OutputRegexp("service/redis created"))
s.T().Log("Redis resources are ready")
```
Do you want to check and block the deployment for the test that might not need Redis under this PR?